src/Console/Shell.php

```diff
@@ -681,6 +681,8 @@ public function error($title, $message = null)
             $this->_io->err($message);
         }
         $this->_stop(1);
+
+        return 1;
```

Comment:
In normal operation, `_stop()` will exit, and this line will never be reached. Is this mostly for tests where `_stop()` has been mocked?